src/style.css

```diff
@@ -0,0 +1,129 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  margin: 0;
+}
+
+.button__text {
+  display: inline;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  margin: 0;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+}
+
+.card__content {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  padding-bottom: 4px;
```

Comment:
use margin-bottom instead of padding-bottom to make outer indent between elements